rush/rush/src/utilities/InstallManager.ts

```diff
@@ -180,7 +176,7 @@ export default class InstallManager {
       console.log(os.EOL + 'Running "npm install" in ' + npmToolFolder);
 
       // NOTE: Here we use whatever version of NPM we happen to find in the PATH
-      Utilities.executeCommandWithRetry('npm', ['install'], MAX_INSTALL_ATTEMPTS, npmToolFolder);
+      Utilities.executeCommandWithRetry('npm', ['install', '--no-optional'], MAX_INSTALL_ATTEMPTS, npmToolFolder);
```

Comment:
>'--no-optional' [](start = 59, length = 15)

This was the outcome of a long discussion.  It will seem odd to someone who's reading the code and trying to understand why we chose that behavior.

Could you add some code comments explaining this design decision?